mautic/mautic
# Fixed email tokens that did not hydrate company data

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Even though company fields showed up in the builder as tokens, i.e. `{contactfield=companyzip}`, these tokens were not replaced with values. This PR fixes this. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create a company and set country zip to 70008.
2. Assign the company to a contact
3. Assign the contact to a segment 
4. Create a segment email and add `{contactfield=companyzip}` to the content. Assign the email to the above segment.
5. Send the email
6. Note that the token is replaced with an empty value.


#### Steps to test this PR:
1. Repeat and the token should be replaced with 70008
2. Run the new tests

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
